docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2916 @@
+. Enter `100`, then click *Save changes*.
+====
+// end::notable-breaking-changes[]
+
+To review the breaking changes in previous versions, refer to the following:
+
+{kibana-ref-all}/8.4/release-notes-8.5.0.html#breaking-changes-8.5.0[8.5.0] | {kibana-ref-all}/8.4/release-notes-8.4.0.html#breaking-changes-8.4.0[8.4.0] | {kibana-ref-all}/8.3/release-notes-8.3.0.html#breaking-changes-8.3.0[8.3.0] | {kibana-ref-all}/8.2/release-notes-8.2.0.html#breaking-changes-8.2.0[8.2.0] | {kibana-ref-all}/8.1/release-notes-8.1.0.html#breaking-changes-8.1.0[8.1.0] | {kibana-ref-all}/8.0/release-notes-8.0.0.html#breaking-changes-8.0.0[8.0.0] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc2.html#breaking-changes-8.0.0-rc2[8.0.0-rc2] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc1.html#breaking-changes-8.0.0-rc1[8.0.0-rc1] | {kibana-ref-all}/8.0/release-notes-8.0.0-beta1.html#breaking-changes-8.0.0-beta1[8.0.0-beta1] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha2.html#breaking-changes-8.0.0-alpha2[8.0.0-alpha2] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha1.html#breaking-changes-8.0.0-alpha1[8.0.0-alpha1]
+
+[float]
+[[features-8.6.0]]
+=== Features
+{kib} 8.6.0 adds the following new and notable features.
+
+Alerting::
+* Notify users by email when assigned to a case {kibana-pull}144391[#144391]
+* Adds flapping state object and interface in AAD index and Event Log {kibana-pull}143920[#143920]
+* Change Alerts > Actions execution order {kibana-pull}143577[#143577]
+* Adds the ability to remove alerts attached to a case {kibana-pull}143457[#143457]
+* This feature allows users to create and close alerts within Opsgenie {kibana-pull}142411[#142411]
+* Adds filter field to index threshold rule type {kibana-pull}142255[#142255]
+* Allow users to see event logs from all spaces they have access to {kibana-pull}140449[#140449]
+
+Elastic Security::
+For the Elastic Security 8.6.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Enterprise Search::
+For the Elastic Enterprise Search 8.6.0 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].
+
+Fleet::
+* Differentiate kubernetes integration multi page experience {kibana-pull}145224[#145224]
+* Prerelease toggle {kibana-pull}143853[#143853]
+* Adds link to skip multi step add integration workflow {kibana-pull}143279[#143279]
+* Request diagnostics {kibana-pull}142369[#142369]
+
+Infrastructure::
+Adds support for the the Unified Search Bar for Query input {kibana-pull}143222[#143222]
+
+Lens & Visualizations::
+Adds support for trend lines in *Lens* metric visualizations {kibana-pull}141851[#141851]
+
+Machine Learning::
+* Trained model testing with index data {kibana-pull}144629[#144629]
+* Adding anomaly score explanations {kibana-pull}142999[#142999]
+
+Management::
+Adds the ability to show sub-feature privileges when using the Basic license {kibana-pull}142020[#142020]
```

Comment:
__note:__ I corrected labels for this one, it should be under `Security` section (added comment below),
```suggestion
```